pages/annonces/[slug].tsx

```diff
@@ -5,36 +5,32 @@ import { ComptesAssociationSection } from '#components/annonces-section/comptes-
 import Meta from '#components/meta';
 import Title from '#components/title-section';
 import { FICHE } from '#components/title-section/tabs';
-import {
-  IAnnoncesAssociation,
-  IAnnoncesBodacc,
-  IComptesAssociation,
-  getAnnoncesFromSlug,
-} from '#models/annonces';
+import { IAnnoncesAssociation, IAnnoncesBodacc } from '#models/annonces';
 import { IAPILoading } from '#models/api-loading';
 import { IAPINotRespondingError } from '#models/api-not-responding';
 import { IUniteLegale, isAssociation } from '#models/index';
+import { getUniteLegaleFromSlug } from '#models/unite-legale';
 import { getCompanyPageDescription, getCompanyPageTitle } from '#utils/helpers';
 import extractParamsFromContext from '#utils/server-side-props-helper/extract-params-from-context';
 import {
   IPropsWithMetadata,
   postServerSideProps,
 } from '#utils/server-side-props-helper/post-server-side-props';
+import { useFetchComptesAssociation, useFetchJOAFE } from 'hooks';
 import { NextPageWithLayout } from 'pages/_app';
 
 interface IProps extends IPropsWithMetadata {
   annoncesAssociation: IAnnoncesAssociation | IAPINotRespondingError;
   bodacc: IAnnoncesBodacc | IAPINotRespondingError | IAPILoading;
-  comptesAssociation: IComptesAssociation | IAPINotRespondingError;
   uniteLegale: IUniteLegale;
 }
 
 const Annonces: NextPageWithLayout<IProps> = ({
-  annoncesAssociation,
-  comptesAssociation,
   metadata: { session },
   uniteLegale,
 }) => {
+  const annoncesAssociation = useFetchJOAFE(uniteLegale);
+  const comptesAssociation = useFetchComptesAssociation(uniteLegale);
```

Comment:
Why not have these two call into the `AnnoncesAssociationSection` and `ComptesAssociationSection` ?